myapp/app/controllers/admin/tasks_controller.rb

```diff
@@ -0,0 +1,75 @@
+class Admin::TasksController < ApplicationController
+  before_action :set_task, only: [:show, :edit, :update, :destroy]
+  before_action :require_login
```

Comment:
`require_admin` works without login, so `require_login` is likely unnecessary here.